common/remote/sockfile.cpp

```diff
@@ -5056,14 +5056,17 @@ class CRemoteFileServer : implements IRemoteFileServer, public CInterface
 
     void run(SocketEndpoint &listenep, bool useSSL)
     {
+        Owned<ISocket> acceptSocket, secureRejectSocket;
```

Comment:
They were necessary to allow the creation of a self contained remote file server.
So that the test could create a socket to be used by the server without conflicting with any existing service.